install.sh

```diff
@@ -83,7 +83,7 @@ configure() {
     args=""
     add_arg() {
         log_debug "Add arg -$1=$2"
-        args="$args -$1=$2"
+        for item in $2; do args="$args -$1=$item"; done
```

Comment:
Nit: use `value` instead of `item`

Reply:
Ok, do you want to squeeze a oneliner for this one or break it down (readability)?

Reply:
I mean the `log_debug` code and arg assign